docs/pages/bssw/bssw_wikize_refs.md

```diff
@@ -43,9 +43,9 @@ at the end of the markdown file, of the form
 - `LAB`: (**REQUIRED**) Alphanumeric *label* for the reference
   - **Note**: The post-processing script renumbers them 1...N
 - `URL`: (**REQUIRED**) The URL for the reference
-- `TIT`: (*optional*) A title for the reference (appears as tool-tip during hover)
-- `{BIB}`: (*optional*) Full bibliographic information for the reference
-  - Here, we are using the `{BIB}` field, embedded in the [link's title](https://www.markdownguide.org/basic-syntax#adding-titles) text as a notational *extension*.
+- `TIT`: (*optional*) A title for the reference (will appear as tool-tip during hover)
+- `{BIB}`: (*optional*) Remaining full bibliographic information for the reference except the title
+  - The `{BIB}` field, embedded in a [markdown reference style link](https://github.github.com/gfm/#reference-link) [title](https://www.markdownguide.org/basic-syntax#adding-titles) is a notational *extension* specific to and known by only the `wikize_refs.py` tool.
```

Comment:
I suggest showing exactly how the reference text is constructed here.  I think it is: `[TIT <br>BIB](URL)`

Reply:
You mean like the examples shown just below this section...

![Screen Shot 2022-12-08 at 3 09 41 PM](https://user-images.githubusercontent.com/5720676/206586174-3cb2eee3-544a-43e4-9735-c1aff71dff39.png)

Or, maybe the template example just above this section...

![Screen Shot 2022-12-08 at 3 10 19 PM](https://user-images.githubusercontent.com/5720676/206586241-3525e927-4971-4f26-bc34-10a49cfa66d0.png)

Reply:
I would be very explicit about it and use the template example.  It is probably good to do this in conjunction with the examples.  I think it will help people understand the nuances of the examples. Thanks